frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-details.tsx

```diff
@@ -46,12 +46,12 @@ const Details: React.FC<DetailsProps> = ({ obj }) => {
           <dl className="co-m-pane__details">
             {size && (
               <>
-                <dt>Size</dt>
-                <dd>{sizeMetrics}</dd>
+                <dt data-test-selector="details-item-label__Size">Size</dt>
+                <dd data-test-selector="details-item-value__Size">{sizeMetrics}</dd>
               </>
             )}
-            <dt>Persistent Volume Claim</dt>
-            <dd>
+            <dt data-test-selector="details-item-label__PVC">Persistent Volume Claim</dt>
```

Comment:
I do not see where the following are used?  If not used, please remove and add back when needed -thanks
```
data-test-selector="details-item-label__Size"
data-test-selector="details-item-value__Size"
data-test-selector="details-item-label__PVC"
```